readthedocs/rtd_tests/tests/test_privacy_urls.py

```diff
@@ -1,7 +1,6 @@
 import re
 
 import mock
-
```

Comment:
@pauloxnet qualche motivo per la rimozione di questi a capo (qui e negli altri file sopra)? Preferirei non toccare il codice upstream

Reply:
In realtà lo riga vuota è assente nel codice RTD (vedi Nota)